fs/api_internal.go

```diff
@@ -170,13 +170,15 @@ func (vS *volumeStruct) inFlightFileInodeDataFlusher(inodeNumber inode.InodeNumb
 }
 
 func (inFlightFileInodeData *inFlightFileInodeDataStruct) inFlightFileInodeDataTracker() {
-	var (
-		flushFirst bool
-	)
 
-	logger.Tracef("fs.inFlightFileInodeDataTracker(): waiting to flush volume '%s' inode %d",
+	logger.Tracef("fs.inFlightFileInodeDataTracker() entry: waiting to flush volume '%s' inode %d",
```

Comment:
Logger already adds the function name to the log message so you should not have to add the function name here.

Reply:
For example,  here is an example log message that I see when I am using this PR:

time="2017-12-11T21:00:11.737919Z" level=info msg="vs.Write(): growing file size 999424 to 1000000  volume 'CommonVolume'  inode 204  offset 999424  len 576" function=Write goroutine=981 package=inode